sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ManifestWriteableProperties.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    public partial class ManifestWriteableProperties
+    {
+        /// <summary> Quarantine state. </summary>
+        internal string QuarantineState { get; set; }
+        /// <summary> Quarantine details. </summary>
```

Comment:
Is there more we can say about these properties?

Reply:
These ones aren't part of the public API.